CMakeLists.txt

```diff
@@ -38,132 +41,23 @@ set(NOVELRT_RESOURCES
 
 file(COPY ${NOVELRT_RESOURCES} DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 
-if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm") #TODO: check if the OPENGL_VERSION flag is actually still needed with the new setup
-    file(GLOB CORECLR lib/CoreCLR/LinuxARM32/*)
-    add_compile_definitions(OPENGL_VERSION=2)
-else()
-    add_compile_definitions(OPENGL_VERSION=3)
-endif()
-
-file(COPY ${CORECLR} DESTINATION lib/CoreCLR/Active)
-
-find_package(Freetype REQUIRED)
-find_package(glad CONFIG REQUIRED)
-find_package(glm CONFIG REQUIRED)
-find_package(SDL2 CONFIG REQUIRED)
-find_package(sdl2-image CONFIG REQUIRED)
-find_package(sdl2-mixer CONFIG REQUIRED)
-find_package(spdlog CONFIG REQUIRED)
-
-# lua doesn't provide native cmake support
-find_path(LUA_INCLUDE_DIR lua.h)
-include_directories(${LUA_INCLUDE_DIR})
-
-find_library(LUA_LIBRARY lua)
-link_libraries(${LUA_LIBRARY})
+include_directories(src)
```

Comment:
Why do we need to search `src` for header files?

Reply:
This is for Doxygen, afaik. I've not touched that logic yet so I just put that line in to ensure it had access (it probably doesn't need it but still).